### PR TITLE
modulesync 5.3.0

### PR DESCRIPTION
**Pull request by jcpunk**
#### Pull Request (PR) description
Update to latest modulesync

#### This Pull Request (PR) fixes the following issues
N/A

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -64,13 +64,6 @@
       "operatingsystemrelease": [
         "18.04"
       ]
-    },
-    {
-      "operatingsystem": "FreeBSD",
-      "operatingsystemrelease": [
-        "12",
-        "13"
-      ]
```
Comment (ekohl) [inline metadata.json:73]: This should be noted somewhere. I think it'd also be good to have it in a separate commit so it isn't lost. Generally speaking we try to keep the modulesync commits completely autogenerated.

File: spec/defines/conf_spec.rb [hunk 2]
```diff
@@ -1,10 +1,12 @@
+# frozen_string_literal: true
+
 require 'spec_helper'
 
 describe 'logrotate::conf' do
-  _, facts = on_supported_os.first
-  let(:facts) { facts }
+  _os, os_facts = on_supported_os.first
```
Comment (ekohl) [inline spec/defines/conf_spec.rb:6]: if `_os` is used, shouldn't it be named `os`?

File: metadata.json (hunk 1 above)
Comment (jcpunk) [reply metadata.json:73]: Updated with second commit saying why

File: spec/defines/conf_spec.rb (hunk 2 above)
Comment (jcpunk) [reply spec/defines/conf_spec.rb]: Done

**Review by ekohl — APPROVED** [review]
I'm going to merge this now since it's a nice compatible change. Master is also failing on the same thing so it's not a regression. We should deal with FreeBSD in a separate PR.
